gbpservice/neutron/services/servicechain/plugins/ncp/node_drivers/nfp_node_driver.py

```diff
@@ -565,6 +577,40 @@ def _wait_for_network_function_delete_completion(self, context,
                       {'network_function': network_function_id})
             raise NodeInstanceDeleteFailed()
 
+    def _create_pt(self, context, ptg_id, name, port_id=None):
+        policy_target = {'name': name,
+                         'description': '',
+                         'tenant_id': context.plugin_context.tenant_id,
+                         'policy_target_group_id': ptg_id,
+                         'port_id': port_id,
+                         'proxy_gateway': False,
+                         'group_default_gateway': False,
+                         'cluster_id': ""}
+        return context.gbp_plugin.create_policy_target(
+            context.plugin_context, {"policy_target": policy_target})
+
+    def _create_policy_target_for_vip(self, context):
```

Comment:
This is *aaS specific code and can't be impl in node_driver. It should either be moved to nfp lbaas plugin/driver or nfp config_orchestrator.